src/utils/user.test.ts

```diff
@@ -7,6 +7,7 @@ import {
   GOVTEAM_DEV,
   GOVTEAM_PROD
 } from 'constants/jobCodes';
+import { Flags } from 'types/flags';
```

Comment:
Not sure if it might be a good idea to change how the `Flags` type is defined; making all its properties optional would avoid the need for typecasts in this file, but it might ripple out through the rest of the code and require more fixes.

Reply:
After thinking about it, I think it's not worth making a change to clean up this test code, given that the feature flags are vital to our application.